automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoTestUtils.scala

```diff
@@ -76,8 +76,7 @@ trait LeonardoTestUtils
   val getAfterCreatePatience = PatienceConfig(timeout = scaled(Span(5, Minutes)), interval = scaled(Span(2, Seconds)))
 
   val multiExtensionClusterRequest = UserJupyterExtensionConfig(
-    nbExtensions = Map("map" -> "gmaps"),
-    combinedExtensions = Map("pizza" -> "pizzabutton")
+    nbExtensions = Map("saturn-iframe-extension" -> "https://app.terra.bio/jupyter-iframe-extension.js")
```

Comment:
shall we still add another `combinedExtensions`?

Reply:
What is the difference between combined Extensions vs regular notebook? Could we use a notebook extension as a combined one?

Reply:
> Could we use a notebook extension as a combined one?
I would give it a try..I don't think there's anything special about `pizzabutton`